### PR TITLE
Added route for amp version of Article

### PR DESCRIPTION
**Pull request by jp-bbc**
_Add a route that will serve an amp version of Article. Have had to increase bundle file size for it to pass in Travis._

* [x] Fixes https://github.com/bbc/simorgh/issues/440
* [ ] Tests added for new features

* [ ] Test engineer approval

### CODE REVIEW

**Comment by sareh** [comment]
Great. You'll also want to add Cypress tests with @jamesbrumpton, so an amp article page can be tested. e.g. `/article/amp/scenario-27`

File: src/app/routes/index.js [hunk 1]
```diff
@@ -6,6 +6,11 @@ const routes = [
     exact: true,
     component: Article,
   },
+  {
+    path: '/article/amp',
```
Comment (sareh) [inline src/app/routes/index.js]: This needs to be `/article/amp/:id` so that you can visit the various articles on the AMP route.

Comment (jp-bbc) [reply src/app/routes/index.js]: Ok. I thought it was a bit odd there wasn't an `:id` path variable. I'll add it in now.

**Review by sareh — APPROVED** [review]
👍 Looks good!

**Review by jtart — APPROVED** [review]

**Pull request merged**